Events

hopeforkoami opened the pull request:
In this branch , I : 
- Copy the files calculate.js and operate.js into a logic/ directory in your project.
- Analyze the files calculate.js and operate.js, I'm sure I understand what they do.
- Import the files in your Calculator component.
- Implement the event handlers I need to use the math logic from calculate.js and operate.js in my React component.
- Test your app in the browser. It is able to perform math operations.

Thanks

hopeforkoami committed:
Add logic folder and his content

hopeforkoami committed:
Handle events

hopeforkoami committed:
Correct Stylelint errors

rustam-tolipov commented on src/components/Calculator.js:
```diff
@@ -1,48 +1,72 @@
 
 // eslint-disable-next-line
 class Calculator extends Component {
+  constructor() {
```
- [ ] You did great job. However, you have some errors showing up. In order to make your project approve. Your all buttons are must be functional, and clickable. You can watch the video and see the errors. 

https://user-images.githubusercontent.com/69011963/198722541-b0719be2-00be-4c0c-903a-a4d30cb30834.mov

rustam-tolipov reviewed (changes requested):
Hello @hopeforkoami, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
_Your project is almost complete!_
_No Linters errors_

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

hopeforkoami committed:
Remarks correction

Taking into account all the comments made by the reviewer
- all buttons are now  functional, and clickable

SajjadAhmad14 reviewed (approved):
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi Konami, 
## To Highlight 🎉
App has a perfect design. :heavy_check_mark:
Git flow is followed. :heavy_check_mark:
## Congratulations 🎉 Thank you for implementing all the requirements 🙏
### Please go ahead and merge the branch!

## Happy Coding 👏👏👏
#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

HAMMAS-SALEEM reviewed (commented):
# Hey @hopeforkoami 👋
# Status: Approved :green_circle: :1st_place_medal: :100:
![success](https://camo.githubusercontent.com/6ada59f6189497c1c2cae6d06f8154b7307a58643f232340e288140ed469eb28/68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966)

Great work on implementing the views. Your project is complete. :blush:
There is nothing else to say about your project. :+1:

## Highlights :1st_place_medal:

✔️ Descriptive PR Title
✔️ Descriptive PR Summary
✔️ Following Linters
✔️ Following requirements

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
